gfsphysics/GFS_layer/GFS_typedefs.meta

```diff
@@ -680,6 +690,7 @@
   dimensions = (horizontal_dimension)
   type = real
   kind = kind_phys
+  active = (flag_for_mass_flux_deep_convection_scheme == flag_for_gf_deep_convection_scheme)
```

Comment:
I am curious how the active is used? Will the variable be allocated if active is false or this attribute will only be used by the ccpp/framework?

Reply:
The attribute is currently only used by the ccpp-framework to perform or skip certain operations in the auto-generated caps. In the future, data entirely internal to physics (i.e. everything in the GFS_interstitital DDT) will be allocated automatically by the ccpp-framework, and then `active` can be used to skip certain allocations.

But for the original/other GFS DDTs in `GFS_typedefs.F90`, it will only be used in the auto-generated caps. That's why the Fortran allocation logic in `GFS_typedefs.F90` needs to be consistent with the `active` attribute in `GFS_typedefs.meta` for those data.